src/modules/ekf2/EKF/ekf.h

```diff
@@ -331,6 +336,7 @@ class Ekf final : public EstimatorInterface
 
 	// get the full covariance matrix
 	const matrix::SquareMatrix<float, State::size> &covariances() const { return P; }
+	float covariance(unsigned r, unsigned c) const { return P(r, c); }
```

Comment:
```suggestion
	float stateCovariance(unsigned r, unsigned c) const { return P(r, c); }
```
Just some nitpicking here.